files/gitignore

```diff
@@ -19,3 +19,13 @@ yarn-error.log
 /package-lock.json.ember-try
 /yarn.lock.ember-try
 /pnpm-lock.ember-try.yaml
+
+<% if (yarn) { %>
+.yarn/*
```

Comment:
Can we add this link here, so people can learn more about these settings. I had to search for this, as a yarn v3 noob... 

```suggestion
# https://yarnpkg.com/getting-started/qa/#which-files-should-be-gitignored
.yarn/*
```